docs/overview.md

```diff
@@ -71,16 +71,18 @@ batteries, which are enumerated below.
   keys to RL/LDK in any way they wish following a simple API. We even support a
   generic API for signing transactions, allowing users to run RL/LDK without any
   private keys in memory and/or putting private keys only on hardware wallets.
-  * [LDK's `KeyManager` docs](https://docs.rs/lightning/0.0.12/lightning/chain/keysinterface/struct.KeysManager.html). While LDK's default implementation is currently within Rust-Lightning, it still is considered a "sample module."
+  * [LDK's `KeysManager` docs](https://docs.rs/lightning/*/lightning/chain/keysinterface/struct.KeysManager.html).
+  While LDK's default implementation is currently within Rust-Lightning, it is
+  still considered a sample module.
 * Transaction filtering
   * Clients running a light client may wish to filter for transactions on a separate server, in which case LDK will tell them about transactions to filter for. More information is available in the [Blockchain Data guide](blockdata.md).
 * Fee estimation
-  * LDK's sample node implementation uses Bitcoin Core's fee estimation API [here](https://github.com/TheBlueMatt/rust-lightning-bitcoinrpc/blob/139a653eeba313284c6d9d2eb2776d30dbb0ca3d/src/chain_monitor.rs#L31).
+  * LDK's sample node implementation uses Bitcoin Core's fee estimation API [here](https://github.com/lightningdevkit/ldk-sample/blob/ddc3431971c51a2cbab375ea3ce8ab6df2caf1dd/src/bitcoind_client.rs#L219-L233).
```

Comment:
This link would be more accurate: https://github.com/lightningdevkit/ldk-sample/blob/main/src/bitcoind_client.rs#L98-L154 that link just retrieves the fees that were already cached.